### ISSUE TITLE
Interesting 

### ISSUE THREAD

**Issue by Mintzero337** [PrinceOfAmber/Cyclic#2415]
Minecraft Version: 1.19.2

Forge Version: 1.253.0-17952

Mod Version: 1.19.2-1.8.4.jar

Single Player or Server: single player

Describe problem (what you were doing; what happened; what should have happened):
While playing a personal mod pack, I attempted to use the build want to plant some crops. Having taken a fully grown crop, I expected it to place a newly planted crop, but instead got the fully grown one. This can be cycled repeatedly to grant infinite crops in a short amount of time, and also functions with modded crops, such as mystic agriculture 

Log file link:

Video/images/gifs (direct upload or link):

**Comment by wuweidmw** [comment]
I would like to ask if there is a Discord, and will there be an update to 1.20.2+?

**Comment by Lothrazar** [comment]
yes the building wands remember blockstates, this seems normal.  1.20.2 will never be supported, but1.20.1 and 1.21.1 and 26.1.x+ will be supported
